### ISSUE TITLE
Ordering of preprocessing steps when including autoreject

### ISSUE THREAD

**Issue by skjerns** [autoreject/autoreject#246]
I'm currently using a dataset for machine learning in which I bandfilter the data at different brain bands. I struggle a bit to understand the best timepoint at which to apply `autoreject` in this case. I have to filter the data before epoching, due to the small epoch length and the relatively long filter bandwidth.

current workflow:

```
1. Downsample data to 100 Hz
2. run ICA
3. Filter data into bands (e.g. 1-4 Hz or 4-8 Hz or 8-12 Hz)
4. Epoch bandfiltered data
```

This is what I thought would probably be the optimal way

```
1. Downsample data to 100 Hz
*. *Epoch equidistant with `tstep=2.0`
*. *run autoreject on these epochs to find bad channels
2. *run ICA with `reject=reject` from bad channels
3. Filter data into bands (e.g. 1-4 Hz or 4-8 Hz or 8-12 Hz)
4. Epoch bandfiltered data
*. run autoreject here on filtered data
```

what would you say is the recommended ordering in this case? Does it even make sense to run `autoreject` after bandfiltering?

**Comment by jasmainak** [comment]
That sounds about right to me. My only comment is about this:

> run autoreject on these epochs to find bad channels

I suppose you mean to find bad epochs? Take a look at this example if you haven't already:

https://github.com/autoreject/autoreject/blob/master/examples/plot_autoreject_workflow.py

**Comment by skjerns** [comment]
Ah, great! I was not aware of the examples folder and this is example is not linked in the documentation :) thanks alot!

btw the [dev docs](https://circleci.com/api/v1.1/project/github/autoreject/autoreject/latest/artifacts/0/html/index.html?branch=master) link doesnt work on the README

**Comment by sappelhoff** [comment]
> btw the dev docs link doesnt work on the README

Thanks for reporting @skjerns - I'll fix that in #247

**Comment by jasmainak** [comment]
Thanks a ton @skjerns and @sappelhoff 

Yes I'm aware of this issue, just need dedicated time to fix it :)
